files/nsswitch.conf

```diff
@@ -0,0 +1,20 @@
+# /etc/nsswitch.conf
+#
+# Example configuration of GNU Name Service Switch functionality.
```

Comment:
Please remove this comment, and instead make the comment describe *why* an nsswitch.conf is provided.

Reply:
Thanks for your advice, the comment is updated.